### ISSUE TITLE
fisher information

### ISSUE THREAD

**Issue by jannaescur** [vrenkens/nabu#14]
just one theoretical curiosity,
why did you decide to have a fisher trainer? In what aspects does retrieving the fisher information changes the training? I don't really know how to interpret the fisher loss. Could you summarize it?

**Comment by vrenkens** [comment]
A student of mine was working on transfer learning between languages using a LAS architecture. A problem with transfer learning is [catastrophic forgetting](https://en.wikipedia.org/wiki/Catastrophic_interference). You can use [elastic weight consolidation](https://arxiv.org/abs/1612.00796) to try and solve this issue. It basically puts a spring on each parameter of the network. If the spring is stiff the parameter is not allowed to deviate far from the initial parameter. The fisher information is used to determine the stiffness of the springs and the fisher loss represents the deviation from the initial model weigted with the stiffness.

**Comment by jannaescur** [comment]
thanks! and I guess that even if using this trainer, the loss given by the test is still the PER?

**Comment by vrenkens** [comment]
yes :)
